nucliadb_sdk/nucliadb_sdk/v2/sdk.py

```diff
@@ -167,7 +167,7 @@ def _request_builder(
     stream_response: bool = False,
     docstring: Optional[docstrings.Docstring] = None,
 ):
-    def _func(self: "NucliaDB", content: Optional[Any] = None, **kwargs):
+    def _func(self: "NucliaDB | NucliaDBAsync", content: Optional[Any] = None, **kwargs):
```

Comment:
sorry, I think the linter is complaining about this line in the CI.

Can you change it to 
```
    def _func(
        self: "NucliaDB | NucliaDBAsync", content: Optional[Any] = None, **kwargs
    ):
``` 
instead?

You can run `make python-code-lint` locally to make sure there are no formatting issues

Reply:
Yes, my bad, updated the code 😃